.circleci/config.yml

```diff
@@ -259,6 +259,13 @@ jobs:
 
   test_ds000246:
     <<: *imageconfig
+    # The resource_class feature allows configuring CPU and RAM resources for
+    # each job. Different resource classes are available for different
+    # executors.
+    #
+    # https://circleci.com/docs/2.0/configuration-reference/#resourceclass
+    resource_class: large
+    no_output_timeout: 15m
```

Comment:
Just an idea -- IIRC the NumPy / SciPy build scripts print something every ~1 min or so while building their code. It helps let people know that nothing is broken during a long processing stage, and has the nice side effect that things like this are not necessary. Any idea if we could add something like this? I assume it's not trivial, in which case feel free to turn this into a new issue.